app.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const fs = require('fs');
+const {join} = require("path");
+const cors = require('cors');
+const logger = require('morgan');
+const createError = require('http-errors');
+const config = require('./config');
+const createSession = require("./middleware/session-middleware");
+// const createSession = require("./services/session");
+const statusRouter = require("./routers/status-endpoints-router");
+const graphqlRouter = require("./routers/graphql-router");
+
+// var authRouter = require('./routes/auth');
+
+
+// print environment variables to log
+console.info(config);
+
+// create logs folder if it does not already exist
+const LOGS_FOLDER = 'logs';
+if (!fs.existsSync(LOGS_FOLDER)) fs.mkdirSync(LOGS_FOLDER);
+
+// create a log write stream in append mode
+const accessLogStream = fs.createWriteStream(join(__dirname, LOGS_FOLDER, 'access.log'), { flags: 'a'});
+
+// initialize the app
+const app = express();
+app.use(cors());
+app.use(logger('combined', { stream: accessLogStream }))
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(express.static(join(__dirname, 'public')));
+
+// add ping and version endpoints
+app.use("/", statusRouter);
```

Comment:
This path will conflict with the route in the crdc-datahub-backend. All endpoints for this service should start with /api/authz